commands/release/release_test.go

```diff
@@ -35,12 +36,95 @@ func TestBreakingChangesSubheaderRegex(t *testing.T) {
 	testRegexPattern(t, "Breaking Changes Subheader", breakingChangesSubheaderRegexStr, validStrings, invalidStrings)
 }
 
-func TestDoBreakingChangesExist(t *testing.T) {
-	noVersion :=
+func Test_parseChangeLogFileNegativeTest(t *testing.T) {
+
+	// test inputs
+	noVersionFound :=
+		`#TBD
+* Something
```

Comment:
Not sure if it matters but we also use dashes in other repos. Should probably work the same, but might be worth it to add into the unit test for completeness.

Reply:
i have added `-` to some of the test case.